about/changelog.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Changelog
+excerpt: Changelog
+keywords: [changelog, upgrades, updates, releases]
+---
+
+# Changelog
+
+## TimescaleDB 2.15.2
+
+June 7, 2024
+
+🐛 Bugfixes
+
+- [#6975](https://github.com/timescale/timescaledb/issues/6975): Fix sort pushdown for partially compressed chunks.
+- [#6976](https://github.com/timescale/timescaledb/issues/6976): Fix removal of metadata function and the update script.
+- [#6978](https://github.com/timescale/timescaledb/issues/6978): Fix segfault in compress_chunk with primary space partition.
+- [#6993](https://github.com/timescale/timescaledb/issues/6993): Disallow hash partitioning on the primary column.
+
+## May 2024 Update
+
+May 31, 2024
+
+### 🔍 Database audit logging via the pg_audit extension
+
+The [PostgreSQL Audit Extension (pgaudit)](https://github.com/pgaudit/pgaudit/) is now available on Timescale. It provides detailed database session and/or object audit logging in the Timescale logs.
+
+If you have strict security and compliance requirements and you need to log all operations on the database level, pgaudit can help. You can also export audit logs to CloudWatch if you like.
+
+To install the extension, simply go to `Service > Operations > Extensions` and search for `pgaudit`. To change the configuration parameters, go to `Database parameters > Advanced parameters` and search for `pgaudit`.
+
+For more details on how to use the extension, please refer to the [pgaudit documentation](https://github.com/pgaudit/pgaudit/).
+
+### 🌡 Timescale now supports the unit extension :thermometer:
```

Comment:
* :thermometer: at the end of the sentence is redundant and not rendered 
* Timescale Cloud now ...
* As a list item like the first title maybe.

Reply:
> 🌡️ at the end of the sentence is redundant and not rendered
Oops, copy paste fail, fixed 9dd2537c3b0c0a91dbfec98dc50826624d4aeffc

> Timescale Cloud now ...
When do we use the word Timescale, and when do we use the word Timescale Cloud? I've seen it used pretty interchangibly, but maybe that's on Slack and not necessarily customer-facing docs. Anyway, I changed it in the sentence 281d0b572cba0d6a3ba1123126499aaa3e6499f3